src/System.Private.ServiceModel/tools/scripts/sync-pr.sh

```diff
@@ -67,26 +68,33 @@ show_banner
 
 # Check parameters
 
-if [ -z "$1" ]; then 
+if [ -z "$1" ]; then
+    show_usage
+    exit 1
+fi 
+
+__repo_id=$1
+
+if [ -z "$2" ]; then 
     show_usage
     exit 1
 else 
-    if [ "$1" == "branch" ] || [ "$1" == "pr" ]; then 
-        __operation_mode=$1
+    if [ "$2" == "branch" ] || [ "$2" == "pr" ]; then 
+        __operation_mode=$2
     else 
         show_usage
         exit 1
     fi 
 fi 
 
-if [ -z "$2" ]; then 
+if [ -z "$3" ]; then
```

Comment:
If you do this at the beginning, will it save you most code to check $1 and $2 and show usage?